io.github.FedericoCalzoni.EventRecorder.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id": "io.github.FedericoCalzoni.EventRecorder",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "base": "com.riverbankcomputing.PyQt.BaseApp",
+    "base-version": "6.7",
+    "cleanup-commands": [
+        "/app/cleanup-BaseApp.sh"
+    ],
+    "command": "EventRecorder.py",
+    "modules": [
+        {
+            "name": "EventRecorder",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/FedericoCalzoni/EventRecorder/archive/refs/tags/v1.0.0.tar.gz",
+                    "dest": "EventRecorder",
+                    "sha256": "a324e76289859a8e26c4a5e9088880ea676578e1f773afffbb8314f001018b2a"
+                }
+            ],
+            "build-commands": [
+                "install -D EventRecorder/EventRecorder/EventRecorder.py /app/bin/EventRecorder.py",
+                "chmod +x /app/bin/EventRecorder.py",
+                "install -D EventRecorder/EventRecorder/logo.png /app/share/icons/hicolor/128x128/apps/io.github.FedericoCalzoni.EventRecorder.png",
+                "install -D EventRecorder/io.github.FedericoCalzoni.EventRecorder.desktop /app/share/applications/io.github.FedericoCalzoni.EventRecorder.desktop"
+            ]
+        }
+    ],
+    "finish-args": [
```

Comment:
this should be above modules.

Reply:
it should be fixed in the commit I have just made